bindings/native_backup_provider.go

```diff
@@ -72,9 +73,17 @@ func (b *NativeBackupProviderBridge) DownloadBackupFiles(nodeID, backupID string
 	return strings.Split(files, ","), nil
 }
 
+func (b *NativeBackupProviderBridge) SetTor(torConfig *tor.TorConfig) {
+	return
```

Comment:
My understanding is that iOS uses the NativeBackupProvider to back up to iCloud. Is there any reason to use Tor when accessing iCloud, assuming that accessing a user's iCloud account over Tor will even work consistently?

Reply:
> We shouldn't use Tor for iCloud, just WebDAV is relevant and only in case the server is using Tor.

That makes sense. WebDAV already works over Tor.